Makefile

```diff
@@ -17,3 +17,28 @@ install:
 start: prep
 	npm start
 
+
+
+all: devel-install build devel-uninstall
+
+# this requires a built source tree and avoids having to install anything system-wide
+devel-install:
+	mkdir -p ~/.local/share/cockpit
+	ln  -sfn $(shell pwd)/cockpit/public $(INSTALL_DIR)
+	mkdir -p pkg/lib
```

Comment:
Hm, this is a bit different than the Cockpit starter kit makefile... do we really need this?
https://github.com/cockpit-project/starter-kit/blob/main/Makefile

Reply:
Haven't had a chance to look at this yet, will come back to it tomorrow :)